app/isomorphic/components/layouts/footer/index.js

```diff
@@ -1,49 +1,31 @@
 import React from "react";
-import PT from "prop-types";
```

Comment:
why are you not using proptypes

Reply:
I started this from scratch so added only what are necessary for now. So i didn't need proptypes yet so i haved added it